python/sqlflow_submitter/tensorflow/input_fn.py

```diff
@@ -53,6 +53,20 @@ def get_dtype(type_str):
         raise TypeError("not supported dtype: %s" % type_str)
 
 
+def tf_generator(gen, selected_cols, feature_column_names, feature_metas):
```

Comment:
The name `tf_generator` is not telling what it does.

Reply:
I will rename it in the next PR.